.circleci/config.yml

```diff
@@ -31,13 +31,17 @@ jobs:
       - run:
           name: Install marquez-airflow package
           command: pip install --user -e .
-      - run: python -m pytest --cov=marquez --cov-report xml ./test
+      - run: python -m pytest --ignore=test/integration --cov=marquez --cov-report xml ./test
```

Comment:
nit: we may want to standardize on `tests` as opposed to `test`, as I've seen the former much more often than the latter. I made this change in the client.

Reply:
done with renaming